### PR TITLE
Adding Public - UPerf Results dashboard to grafonnet.

### PR DESCRIPTION
**Pull request by Manda-supraja26**
## Type of change

- [ ] Refactor
- [X] New feature
- [ ] Bug fix
- [X] Optimization
- [ ] Documentation Update

## Description

Adding Public - UPerf Results dashboard to grafonnet.
## Related Tickets & Documents

- Related Issue #
- Closes #

## Checklist before requesting a review

- [X] I have performed a self-review of my code.
- [ ] If it is a core feature, I have added thorough tests.

## Testing
- Please describe the System Under Test.
- Please provide detailed steps to perform tests related to this code change.
- How were the fix/results from this change verified? Please provide relevant screenshots or results.

### CODE REVIEW

**Comment by Manda-supraja26** [comment]
@vishnuchalla  

old dashboard: http://dittybopper-dittybopper.apps.smanda-ocp1.perfscale.devcluster.openshift.com/d/noO0pWhSk/public-uperf-results?orgId=1

new dashboard: http://dittybopper-dittybopper.apps.smanda-ocp1.perfscale.devcluster.openshift.com/d/caF0pZ2Sk/public-uperf-results-dashboard?orgId=1&refresh=30s

**Comment by vishnuchalla** [comment]
> @vishnuchalla
> 
> old dashboard: http://dittybopper-dittybopper.apps.smanda-ocp1.perfscale.devcluster.openshift.com/d/noO0pWhSk/public-uperf-results?orgId=1
> 
> new dashboard: http://dittybopper-dittybopper.apps.smanda-ocp1.perfscale.devcluster.openshift.com/d/caF0pZ2Sk/public-uperf-results-dashboard?orgId=1&refresh=30s

Hosting these dashboards on your own cluster is not useful when they have `elasticsearch` as data source. Can you use your dev grafana instance instead to host them for testing and PR review purposes please?

**Comment by Manda-supraja26** [comment]
old dashboard: https://grafana.rdu2.scalelab.redhat.com:3000/d/bca19562-2624-4102-bdd1-da7ac8561532/public-uperf-results-old?orgId=1

new dashboard: https://grafana.rdu2.scalelab.redhat.com:3000/d/b07cef83-826a-4f87-819b-0cd8529743ef/public-uperf-results-dashboard-new?orgId=1&refresh=30s

File: templates/General/uperf-v2.jsonnet [hunk 1]
```diff
@@ -0,0 +1,32 @@
+local panels = import '../../assets/uperf/panels.libsonnet';
+local queries = import '../../assets/uperf/queries.libsonnet';
+local variables = import '../../assets/uperf/variables.libsonnet';
+local g = import 'github.com/grafana/grafonnet/gen/grafonnet-latest/main.libsonnet';
+
+g.dashboard.new('Public - UPerf Results dashboard')
++ g.dashboard.withTags(['network', 'performance'])
++ g.dashboard.time.withFrom('now/y')
```
Comment (vishnuchalla) [inline templates/General/uperf-v2.jsonnet:8]: This seems to incorrect according to [here](https://github.com/cloud-bulldozer/performance-dashboards/blob/master/templates/General/uperf-perf.jsonnet#L235).

Comment (Manda-supraja26) [reply templates/General/uperf-v2.jsonnet:8]: yes, my bad i will correct it

**Comment by vishnuchalla** [comment]
> old dashboard: https://grafana.rdu2.scalelab.redhat.com:3000/d/bca19562-2624-4102-bdd1-da7ac8561532/public-uperf-results-old?orgId=1
> 
> new dashboard: https://grafana.rdu2.scalelab.redhat.com:3000/d/b07cef83-826a-4f87-819b-0cd8529743ef/public-uperf-results-dashboard-new?orgId=1&refresh=30s

I just took a look at both the dashboards side by side, Values in `UPerf Result Summary --> message_size` column are completely irrelevant. Can you please cross check everything again and correct them?

**Comment by Manda-supraja26** [comment]
image

here in the old dashboard the `message_size ` is mentioned exact number. but in the new one it is rounding off to value and giving the B/s. `16384 is 16.4k (16400).`

**Comment by vishnuchalla** [comment]
> image
> 
> here in the old dashboard the `message_size ` is mentioned exact number. but in the new one it is rounding off to value and giving the B/s. `16384 is 16.4k (16400).`

Yes can you please correct that? Also message size in Kb/s doesn't make sense.

**Comment by suprajamanda** [comment]
please refer these links
old dashboard:  https://grafana.rdu2.scalelab.redhat.com:3000/d/f97539fd-8528-43c6-8fa4-ab34d81db679/public-uperf-results-old?orgId=1

new dashboard: https://grafana.rdu2.scalelab.redhat.com:3000/d/c592a5b3-2b88-45a2-acfc-85cd3a1dae6b/public-uperf-results-dashboard-new?orgId=1

**Review by vishnuchalla — APPROVED** [review]
lgtm. Thanks :)

**Pull request merged**